profiles.go

```diff
@@ -6,61 +6,72 @@ import (
 	"strings"
 )
 
+// CommunityVisibilityState contains the visibility of the user
 type CommunityVisibilityState int
 
 const (
-	Private     CommunityVisibilityState = 1
-	FriendsOnly                          = 2
-	Public                               = 3
+	// Private community visibility state
+	Private CommunityVisibilityState = 1
+	// FriendsOnly community visibility state
+	FriendsOnly = 2
```

Comment:
Now that I see it, do `FriendsOnly` and `Public` have the `CommunityVisibilityState` type? Wouldn't they be untyped like the `Relationship` constant before you fixed it above in `friends.go`?